frontend/tests/controllers/ContestCloneTest.php

```diff
@@ -106,6 +106,41 @@ public function testCreateContestCloneWithTheSameAlias() {
             $this->assertEquals('aliasInUse', $e->getMessage());
         }
     }
+    /**
+     * A PHPUnit data provider for all the plagiarism threshold values in a
+     * contest.
+     *
+     * @return list<array{0:bool, 1:int}>
+     */
+    public function plagiarismThresholdProvider(): array {
+        return [
+            [true, 90],
+            [false, 0],
+        ];
+    }
+    /**
+     * Check if the plagiarism value is stored correctly in the database.
```

Comment:
```suggestion

    /**
     * Check if the plagiarism value is stored correctly in the database.
```
just for consistency between functions